packages/@okta/vuepress-site/docs/release-notes/2025-okta-identity-engine/index.md

```diff
@@ -8,6 +8,27 @@ title: Okta Identity Engine API release notes 2025
 
 ## November
 
+### Weekly release 2025.11.3
+
+| Change | Expected in Preview Orgs |
+|--------|--------------------------|
+| [Bugs fixed in 2025.11.3](#bugs-fixed-in-2025-11-3)| December 3, 2025 |
+
+
+#### Bugs fixed in 2025.11.3
+
+* The Create Group Push Mapping endpoint (`POST /api/v1/apps/{appId}/group-push/mappings`) didn't return validation errors when requests included null attributes or incompatible Active Directory configurations. (OKTA-1061899)
```

Comment:
Nit: This endpoint method title is in sentence case in Redocly. I'm not sure if you want to capitalize the "C"?
```suggestion
* The Create a group push mapping endpoint (`POST /api/v1/apps/{appId}/group-push/mappings`) didn't return validation errors when requests included null attributes or incompatible Active Directory configurations. (OKTA-1061899)
```